gem-push-action

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/bash 
+set -e -o pipefail
+
+if parse-gemspec --is-pre-release; then
+    if [[ $INPUT_PRE_RELEASE != true ]]; then
+    echo Ignoring pre-release. Set input pre-release: true, to release
```

Comment:
Move the ', to release' to the front: To release, pass `pre-release: true` as an input.